stringtie/1.2.4/prepDE.py

```diff
@@ -118,59 +126,51 @@ def t_overlap(t1, t2): #from badGenes: chromosome, strand, cluster, start, end,
         my_ID=opts.key+str((u+1))
         print my_ID
         legend.append(list(itertools.chain.from_iterable([[my_ID],c]))) #my_ID, clustered transcript IDs
-#        print c
         for t in c:
-            geneIDs[t]=my_ID            
-##            geneIDs[t]="|".join(c) #duct-tape transcript IDs together, disregarding ref_gene_names and things like that
+            geneIDs[t]=my_ID
 
     print legend
-    
+
     with open(opts.legend, 'w') as l_file:
         my_writer=csv.writer(l_file)
         my_writer.writerows(sortedClusters)
-    
+
 geneDict={} #key=gene/cluster, value=dictionary with key=sample, value=summed counts
 t_dict={}
 for q, s in enumerate(samples):
     print q,s
 
-    try:
-        with open(glob.iglob(os.path.join(opts.input,s,"*.gtf")).next()) as f: #grabs first .gtf file it finds inside the sample subdirectory
-
-##        split=[t[:len(t)-1]+t[len(t)-1].split(";") for t in split]
-##        split=[t[:len(t)-1] for t in split] #eliminate '\n' at end
-##        split=[[e.lstrip() for e in t] for t in split]
-        #should consider making stuff into dictionaries, maybe each split line
```

Comment:
Removed some more commented out code lines. The actual comment that was removed is more of a to-do note intended for the script maintainers, so I removed it too.